src/dbnode/storage/bootstrap/bootstrapper/peers/source.go

```diff
@@ -90,9 +84,85 @@ func (s *peersSource) AvailableData(
 	return s.peerAvailability(nsMetadata, shardsTimeRanges, runOpts)
 }
 
-func (s *peersSource) ReadData(
+func (s *peersSource) AvailableIndex(
 	nsMetadata namespace.Metadata,
 	shardsTimeRanges result.ShardTimeRanges,
+	runOpts bootstrap.RunOptions,
+) (result.ShardTimeRanges, error) {
+	if err := s.validateRunOpts(runOpts); err != nil {
+		return nil, err
+	}
+	return s.peerAvailability(nsMetadata, shardsTimeRanges, runOpts)
+}
+
+func (s *peersSource) Read(
+	namespaces bootstrap.Namespaces,
+) (bootstrap.NamespaceResults, error) {
+	results := bootstrap.NamespaceResults{
+		Results: bootstrap.NewNamespaceResultsMap(bootstrap.NamespaceResultsMapOptions{}),
+	}
+
+	// NB(r): Perform all data bootstrapping first then index bootstrapping
+	// to more clearly deliniate which process is slower than the other.
+	nowFn := s.opts.ResultOptions().ClockOptions().NowFn()
+	start := nowFn()
+	s.log.Info("bootstrapping time series data from peers start")
```

Comment:
nit: remove references to `peers` from these logs since it's part of the logger?

Reply:
Sounds good.